### DISCUSSION TITLE
tun 模式 CPU占用问题

### DISCUSSION THREAD

**Discussion by shadowmimosa** [MetaCubeX/mihomo#1505] [Q&A]
开了tun模式，刚发现CPU占用有点异常
排查发现就是迅雷在下载的时候一堆 warning
![image](https://github.com/user-attachments/assets/364c73e3-953b-4e2f-8d02-d420895d1dcc)
日志写了一个多G
![image](https://github.com/user-attachments/assets/48cae833-95b6-4872-b24c-31397102c708)
都是这种超时或者拒绝响应的报错
![image](https://github.com/user-attachments/assets/cc359b74-809e-49c7-8f2d-9b4b40e853f0)

有啥方法避免这种问题

**Comment by xiandu999-yu** [comment]
BT流量？
